server/routes/hash-password.js

```diff
@@ -0,0 +1,27 @@
+const bcrypt = require('bcrypt');
+
+// passwords to hash of previously added users
+const usersToHash = [
+    { username: 'ahuynh', password: '' },
+    { username: 'coolcat', password: '' }
+];
```

Comment:
Yes, this was for a function to hash previous accounts that were added to the database before implementing hashing the passwords. I hashed the passwords for those two accounts using the function in this file. But I think I will delete this file since it will no longer be needed, since passwords will be hashed when created through the registration form.

Reply:
Removed file with commit: [d89288c](https://github.com/abbiehuynh/my-crochet-corner/pull/4/commits/d89288c3e6c68626b71457d6378eb02408fb556b)